odrive_serial_test/odrive_serial_test.ino

```diff
@@ -0,0 +1,58 @@
+#include <ODriveArduino.h>
+#include <HardwareSerial.h>
+
+HardwareSerial& odrive_serial = Serial2;
+
+ODriveArduino odrive(odrive_serial);
```

Comment:
So this script is for only one odrive at a time, correct?

Reply:
Yes @agupta231 . Andrew only has one Odrive so this makes sense

Reply:
Well it's just one also because I was experimenting. You don't wanna have an experimentation script running multiple odrives (in most cases)